packages/components/src/menu-items-choice/index.tsx

```diff
@@ -0,0 +1,50 @@
+/**
+ * WordPress dependencies
+ */
+import { check } from '@wordpress/icons';
+
+/**
+ * Internal dependencies
+ */
+import MenuItem from '../menu-item';
+import type { MenuItemsChoiceProps } from './types';
+
+const noop = () => {};
+
+function MenuItemsChoice( {
+	choices = [],
+	onHover = noop,
+	onSelect,
+	value,
+}: MenuItemsChoiceProps ) {
+	return (
+		<>
+			{ choices.map( ( item ) => {
+				const isSelected = value === item.value;
+				return (
+					<MenuItem
+						key={ item.value }
+						role="menuitemradio"
+						icon={ isSelected && check }
+						info={ item.info }
+						isSelected={ isSelected }
+						shortcut={ item.shortcut }
+						className="components-menu-items-choice"
+						onClick={ () => {
+							if ( ! isSelected ) {
+								onSelect( item.value );
+							}
+						} }
+						onMouseEnter={ () => onHover( item.value ) }
+						onMouseLeave={ () => onHover() }
```

Comment:
Any reason why this was changed from `() => onHover( null )`? This could be a breaking change for a consumer that was specifically doing `null` checks, so I think we should avoid the change if possible.

Reply:
I understand we need to keep the API to make sure we don't introduce a breaking change at this point. There's no particular reason other than me finding this kind of an odd API to return explicitly `null` in case no element is hovered. Reverted via https://github.com/WordPress/gutenberg/pull/47180/commits/9baa7fb26a56a3c386f8fa70f49ad3fb95d8eb4e.